app/javascript/components/challenges/Counter.tsx

```diff
@@ -3,23 +3,40 @@ import * as React from 'react';
 import NextButton from './NextButton';
 
 export default function Counter() {
-  const [count, setCount] = React.useState<number>(0);
+  const [count, setCount] = React.useState<number>(-10);
+
+  const [reset, setReset] = React.useState<number>(0);
 
   return (
     <div>
       <h1>🥉 Challenge 1</h1>
       <h3>1. Fix the counter.</h3>
       <h3>2. Add a way to reset the counter.</h3>
       <h3>3. Set the counter to -10.</h3>
+      <h3>4. The default reset value is 0</h3>
 
       <div>
         <input readOnly value={count} />
       </div>
       <div>
-        <button onClick={() => setCount(count ** 2)}>Decrease</button>
+        <button onClick={() => setCount(count - 1)}>Decrease</button>
         <button onClick={() => setCount(count + 1)}>Increase</button>
+        <button onClick={() => setCount(reset)}>Reset</button>
+        <button onClick={() => setReset(count)}>Set Current as Reset Value</button>
```

Comment:
All looks good. Minor point but ideally you shouldn't use inline anonymous functions i.e. these should be defined above and then used e.g
```
const handleDecrease = () => setCount(count - 1)

<button onClick={handleDecrease}>Decrease</button>
```

It just stops the function being redefined on each re-render